openapi/platform/platform.yaml

```diff
@@ -5165,6 +5165,11 @@ components:
                 - `fulfilled`: The order and its items have been successfully processed. This includes activating the products with the vendor(s).
                 - `error`: One or more of the orders items failed to be processed
                 - `archived`: The order has been marked as archived
+            tags:
+              type: array
+              description: A tag is a label that can be used for grouping and filtering your list of accounts. They may be any string upto 30 characters in length.
+              items:
+                type: string
```

Comment:
The comment mentions a limit of 30 characters. We should probably also set that property here.

Reply:
Property added.